internal/convert.go

```diff
@@ -85,13 +85,16 @@ const (
 	Widened
 	Time
 	StringOverflow
+	IllegalName
 )
 
 // NameAndCols contains the name of a table and its columns.
 // Used to map between source DB and Spanner table and column names.
 type NameAndCols struct {
```

Comment:
This works but I am leaning towards avoiding channging the strucutre of the NameAndCols struct mainly because it will change the session file that will get exported and hence can cause backwards compatibility issues. An easy workaround is to add another field in conv similar to names and conv but for fks and indexes.

Reply:
ohkay, will do.